From b13d641f85cf93b5e4457709b4d3efcb9e665a20 Mon Sep 17 00:00:00 2001
From: Salini Venate <salini.venate@intel.com>
Date: Wed, 28 Jun 2023 18:58:45 +0530
Subject: [PATCH] Fix VINTF errors

Removing below VINTF entries from device/framework manifest
as they are incuded in the respective service modules.
	android.hardware.security.keymint
	android.hardware.security.secureclock
	android.hardware.security.sharedsecret
	android.hardware.health
	android.hardware.boot

Tracked-On: NA
Signed-off-by: Salini Venate <salini.venate@intel.com>
---
 .../celadon_ivi/framework_manifest.xml        |  9 ----
 .../device-specific/celadon_ivi/manifest.xml  | 44 -------------------
 2 files changed, 53 deletions(-)

diff --git a/groups/device-specific/celadon_ivi/framework_manifest.xml b/groups/device-specific/celadon_ivi/framework_manifest.xml
index acbd3c2..6d0ce85 100644
--- a/groups/device-specific/celadon_ivi/framework_manifest.xml
+++ b/groups/device-specific/celadon_ivi/framework_manifest.xml
@@ -166,15 +166,6 @@
             <instance>external/0</instance>
         </interface>
     </hal>
-    <hal format="hidl">
-        <name>android.hardware.boot</name>
-        <transport>hwbinder</transport>
-        <version>1.2</version>
-        <interface>
-            <name>IBootControl</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <hal format="hidl">
         <name>android.hardware.renderscript</name>
         <transport arch="32+64">passthrough</transport>
diff --git a/groups/device-specific/celadon_ivi/manifest.xml b/groups/device-specific/celadon_ivi/manifest.xml
index e834dd4..60f55b8 100644
--- a/groups/device-specific/celadon_ivi/manifest.xml
+++ b/groups/device-specific/celadon_ivi/manifest.xml
@@ -4,41 +4,6 @@
 {{#ota-update}}
 <manifest version="1.0" type="device">
 {{/ota-update}}
-    <hal format="aidl">
-        <name>android.hardware.security.keymint</name>
-        <version>2</version>
-        <interface>
-            <name>IKeyMintDevice</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-    <hal format="aidl">
-        <name>android.hardware.security.keymint</name>
-        <version>2</version>
-        <interface>
-            <name>IRemotelyProvisionedComponent</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-    <hal format="aidl">
-        <name>android.hardware.security.secureclock</name>
-        <interface>
-            <name>ISecureClock</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-    <hal format="aidl">
-        <name>android.hardware.security.sharedsecret</name>
-        <interface>
-            <name>ISharedSecret</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-    <hal format="aidl">
-        <name>android.hardware.health</name>
-        <version>1</version>
-        <fqname>IHealth/default</fqname>
-    </hal>
     <hal format="hidl">
         <name>android.hardware.thermal</name>
         <transport>hwbinder</transport>
@@ -115,15 +80,6 @@
             <instance>legacy/0</instance>
         </interface>
     </hal>
-    <hal format="hidl">
-        <name>android.hardware.boot</name>
-        <transport>hwbinder</transport>
-        <version>1.2</version>
-        <interface>
-            <name>IBootControl</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <hal format="hidl">
         <name>android.hardware.renderscript</name>
         <transport arch="32+64">passthrough</transport>
-- 
2.17.1

